src/components/card/card.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import Box from "@mui/joy/Box";
+import Card from "@mui/joy/Card";
+import CardCover from "@mui/joy/CardCover";
+import CardContent from "@mui/joy/CardContent";
+import Typography from "@mui/joy/Typography";
+import PersonIcon from "@mui/icons-material/Person";
+import LocationOnRoundedIcon from "@mui/icons-material/LocationOnRounded";
+import HelpIcon from "@mui/icons-material/Help";
+import { FC } from "react";
+import { SearchResult } from "../../models/request/search";
+
+interface CardResultsProps {
+    content: SearchResult;
+}
+export const CardComponent: FC<CardResultsProps> = ({ content }) => {
+    return (
+        <Link to={`/${content.uriPart}`}>
```

Comment:
Maybe have `/character/zeus` instead of zeus? Otherwise, i think our routing will be hard to define.

Reply:
Agree. On it...

Reply:
Done :)